fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/DataServlet.java

```diff
@@ -471,9 +468,9 @@ private void setupAsynchronousWorkflows(
         // resources once per subscription. A connectable Observable's chain is only executed once
         // regardless of the number of subscriptions.
         ConnectableObservable<Either<PreResponse, JobRow>> payloadEmitter;
-        if (asyncAfter == DataApiRequest.ASYNCHRONOUS_ASYNC_AFTER_VALUE) {
+        if (asyncAfter == DataApiRequestImpl.ASYNCHRONOUS_ASYNC_AFTER_VALUE) {
```

Comment:
You shouldn't be assuming a specific implementation of the interface, so these constants probably need to be moved to the interface.